pom.xml

```diff
@@ -86,12 +86,11 @@
 
   <properties>
     <checkstyle.version>8.11</checkstyle.version>
-    <sonar.version>5.6.6</sonar.version>
-    <sonar-java.version>3.7</sonar-java.version>
+    <sonar.version>7.2</sonar.version>
```

Comment:
can we keep it as 6.7 ?
6.7 is LTS - https://www.sonarqube.org/downloads/

it is completely ok to test it additionally on 7.X and try to be compatible.

Reply:
Wercker is upgraded to use 6.7.5 version of sonar.

Is it possible to keep code compatible with 6.7 and 7.3 ?

I am still searching ability to run 7.3+ in CI as as verification that we are compatible with LST version, and kind of predict when we need to upgrade.